docs/src/parameter_distributions.md

```diff
@@ -31,6 +36,21 @@ And the distribution is created by calling:
 prior = ParameterDistribution(distribution,constraint,name)
 ```
 
+```@setup zero_point_seven
+using Distributions 
+using Plots 
+N=50 
+x_eval = collect(-10:20/400:10) 
+#bounded in [0.0,1.0]
+unconstrained_to_constrained(x) = (1 * exp(x) + 0.0) / (exp(x) + 1)
+dist= pdf(Normal(1,0.5),x_eval) 
+constrained_x_eval = unconstrained_to_constrained.(x_eval) 
+```
+The push-forward of the pdf and the push-forward of the mean look like
```

Comment:
Maybe add the original normal distribution to this plot, to highlight that the push-forward maps from an unconstrained to a constrained distribution?

Reply:
done. please have a look